src/scripts/quests/QuestLineHelper.ts

```diff
@@ -318,6 +318,39 @@ class QuestLineHelper {
         App.game.quests.questLines().push(plasmaUnovaQuestLine);
     }
 
+    public static createSkullAetherAlolaQuestLine() {
+        const skullAetherAlolaQuestLine = new QuestLine('Eater of Light', 'A dangerous Pokémon from another world threatens the Alola region.');
+
+        const clearAetherParadiseGym = new CustomQuest(1, 0, 'A strange creature has appeared in Aether Paradise. Make it go away. Clear the Aether Paradise gym.', () => App.game.statistics.gymsDefeated[GameConstants.getGymIndex('Aether Paradise')](), App.game.statistics.gymsDefeated[GameConstants.getGymIndex('Aether Paradise')]());
```

Comment:
Do we need the extra `gymsDefeated` at the end for this?
It should automatically be based on your current progress (based on the value that the focus returns)
```suggestion
        const clearAetherParadiseGym = new CustomQuest(1, 0, 'A strange creature has appeared in Aether Paradise. Make it go away. Clear the Aether Paradise gym.', () => App.game.statistics.gymsDefeated[GameConstants.getGymIndex('Aether Paradise')]());
```

Reply:
NGL I just copied it down from the Plasma one.